activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/service/ProjectServiceImpl.java

```diff
@@ -452,4 +447,31 @@ public void validateProject(Project project) {
 
         return validationErrors.stream();
     }
+
+    private Project importModelsFromProjectHolder(ProjectHolder projectHolder, String id){
```

Comment:
I would get rid of this 'id' and log the error in the controller using this feature.

Reply:
Removed